### PR TITLE
Iliadbox

### PR DESCRIPTION
**Pull request by lucanastasio**

### CODE REVIEW

File: _router_pon/free_iliad_F-MDCONU3A.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+| 2.5/5/10GBaseT   | No?           				|
+| Optics           | SFP	                             |
+| IP address       | ?                                       |
+| Web Gui          | ?                              	     |
+| SSH              | ?					     |
+| Telnet           | ?					     |
+| Serial           | ✅                                       |
+| Serial baud      | 57600                                   |
+| Serial encoding  | 8-N-1                                   |
+| Form Factor      | ONT		                     |
```
Comment (simonebortolin) [inline _router_pon/free_iliad_F-MDCONU3A.md:39]:
```suggestion
| Type      | router with SFP w/o MAC support |
```

File: _router_pon/free_iliad_F-MDCONU3A.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+## Reading all EEPROM
+
+# EEPROM
+
+There is an SFP plug on the UNI side with an embedded EEPROM.
+
+## EEPROM layout
+
+```
```
Comment (simonebortolin) [inline _router_pon/free_iliad_F-MDCONU3A.md:180]: would be to table like ma5671a/zyxel pgm3000/fs

File: _router_pon/free_iliad_box_pop.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+|                  |                                         |
+| ---------------- | --------------------------------------- |
+| Vendor/Brand     | Free/Iliad                              |
+| Model            |                                         |
+| ODM              |                                         |
+| ODM Product Code |                                         |
+| Chipset          | BCM63153                                |
+| Flash            | KLM4G1FETE-B041 (4GB eMMC 5.1)          |
+| RAM              | 2x NT5CC128M16JR-EK (512MB DDR3L-1866)  |
```
Comment (simonebortolin) [inline _router_pon/free_iliad_box_pop.md:18]: put size first and then other information

File: _router_pon/free_iliad_box_pop.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+| CPU              |                                 |
+| CPU Clock        |                                         |
+| Bootloader       |                                         |
+| System           |                                         |
+| Load addr        |                                         |
+| HSGMII           |                                         |
```
Comment (simonebortolin) [inline _router_pon/free_iliad_box_pop.md:24]:
```suggestion
| HSGMII           | No |
```

**Comment by simonebortolin** [comment]
Then please change this page

https://github.com/lucanastasio/hack-gpon.github.io/blob/iliadbox/_sfp/ont-wo-mac.md

File: assets/img/iliad/onu1/BCM55030_features.jpg
Comment (simonebortolin) [inline assets/img/iliad/onu1/BCM55030_features.jpg:1]: image

this photo has a higher quality

**Comment by nanomad** [comment]
Reopened

**Comment by nanomad** [comment]
Reopened

**Comment by simonebortolin** [comment]
#240

**Pull request closed without merging**